sovrin_client/test/agent/test_helper.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import logging
+import time
+import os
+
+from sovrin_client.test.agent.bulldog_helper import bulldogLogger
+from sovrin_common.config import agentLoggingLevel
+
+
+def test_loggerLevelSuccess(tdir):
```

Comment:
I think you are mixing two conventions in the test function names. You are using underscore to separate words and also using camel case. I am not 100% sure, but I guess where we landed was camel case.